front/src/store/index.js

```diff
@@ -1,18 +1,19 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 import сelebrities from './modules/celebrities.store.js';
+import movies from './modules/movies.store.js';
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: {
-    baseUrl: 'https://localhost:5001'
+    baseUrl: 'https://localhost:44333'
```

Comment:
don`t you think that other team members could use other port? when you change somethings you should understand why you do it, why it was in previous state and how to better do it

please, relocate it to dev.config.json, change baseUrl from const to value from this config and commit. in next commit add this config to gitignore to make this file just local for each team member